01-node-tutorial/answers/customEmitter.js

```diff
@@ -0,0 +1,30 @@
+const EventEmitter = require('events')
+
+const dataReadyEmitter = new EventEmitter()
+const dataAcceptedEmitter = new EventEmitter()
+const dataFetchedEmitter = new EventEmitter()
```

Comment:
You could also have a single `dataEventsEmitter` and define all the event handling and emission on that single object.